modules/pathtracing/include/inviwo/pathtracing/uniformgrid3d.h

```diff
@@ -0,0 +1,195 @@
+/**********************************************************************
+ * Copyright (C) 2015 Vistinct AB
+ * All Rights Reserved.
+ *
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * No part of this software may be reproduced or transmitted in any
+ * form or by any means including photocopying or recording without
+ * written permission of the copyright owner.
+ *
+ * Primary author : Daniel Jönsson
+ *
+ **********************************************************************/
```

Comment:
This should not be in here, remember that you have signed the inviwo CLA. 
Also this class seems pointless, just use a volume.